### ISSUE TITLE
Porting to different OS(s)

### ISSUE THREAD

**Issue by Lukas-Batema** [thebaselab/codeapp#207]
I would love to see this, as well as strongly suggest porting this to macOS and Android. It would build up the community!

**Comment by bummoblizard** [comment]
I don't see how it makes sense on macOS when there's [VS Code](https://github.com/microsoft/vscode). 😂

**Comment by Lukas-Batema** [comment]
> I don't see how it makes sense on macOS when there's [VS Code](https://github.com/microsoft/vscode). 😂

true, i was thinking by enabling iCloud, to allow switching devices.

**Comment by bummoblizard** [comment]
> > I don't see how it makes sense on macOS when there's [VS Code](https://github.com/microsoft/vscode). 😂
> 
> true, i was thinking by enabling iCloud, to allow switching devices.

It's already possible to open folders located on iCloud in Code App. The same goes for VS Code on Mac.

**Comment by Lukas-Batema** [comment]
> > > I don't see how it makes sense on macOS when there's [VS Code](https://github.com/microsoft/vscode). 😂
> > 
> > 
> > true, i was thinking by enabling iCloud, to allow switching devices.
> 
> It's already possible to open folders located on iCloud in Code App. The same goes for VS Code on Mac.

hmm, i cannot get it to work. setting the apps folder in icloud, i mean

**Comment by bummoblizard** [comment]
> > > > I don't see how it makes sense on macOS when there's [VS Code](https://github.com/microsoft/vscode). 😂
> > > 
> > > 
> > > true, i was thinking by enabling iCloud, to allow switching devices.
> > 
> > 
> > It's already possible to open folders located on iCloud in Code App. The same goes for VS Code on Mac.
> 
> hmm, i cannot get it to work. setting the apps folder in icloud, i mean

Click the Open Folder button located in explorer or welcome page and choose a directory located in iCloud Drive.

**Comment by Lukas-Batema** [comment]
> > > > > I don't see how it makes sense on macOS when there's [VS Code](https://github.com/microsoft/vscode). 😂
> > > > 
> > > > 
> > > > true, i was thinking by enabling iCloud, to allow switching devices.
> > > 
> > > 
> > > It's already possible to open folders located on iCloud in Code App. The same goes for VS Code on Mac.
> > 
> > 
> > hmm, i cannot get it to work. setting the apps folder in icloud, i mean
> 
> Click the Open Folder button located in explorer or welcome page and choose a directory located in iCloud Drive.

Ah, ok.

**Comment by Lukas-Batema** [comment]
I should keep this open, at least for porting to android...

**Comment by Lukas-Batema** [comment]
@bummoblizard Is this going to ever be implemented, or may I close the issue?
